build/build-image/cross/Dockerfile

```diff
@@ -26,8 +25,9 @@ ENV KUBE_CROSSPLATFORMS \
 RUN cd /usr/src/go/src && for platform in ${KUBE_CROSSPLATFORMS}; do GOOS=${platform%/*} GOARCH=${platform##*/} ./make.bash --no-clean; done
 
 # Install g++, then download and install protoc for generating protobuf output
-RUN apt-get install -y g++ && apt-get clean && rm -rf /var/lib/apt/lists/* &&\
-    mkdir -p /usr/local/src/protobuf && cd /usr/local/src/protobuf &&\
+RUN apt-get update && apt-get install -y g++ rsync && apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
rm -rf /var/lib/apt/lists/* doesn't reduce the size of the image unless it's in the same layer (on the same line) as apt-get update. I'm trying to keep this image small to reduce build times from a clean environment.